src/dmd/globals.d

```diff
@@ -561,15 +561,19 @@ struct Loc
     const(char)* filename; // either absolute or relative to cwd
     uint linnum;
     uint charnum;
+    version(ExtraLocation)
```

Comment:
I agree with Razvan. We should have a more generic name of the version identifier that works for all additions that are specific to DMD as a library.